src/components/summary/index.scss

```diff
@@ -3,3 +3,7 @@
     width: 100%;
   }
 }
+
+.new-cases {
```

Comment:
remove this, just use semantic-ui predefined style should be fine.
doc: https://react.semantic-ui.com/collections/table/#states-positive-negative